bing.com.bingwebmasterdpoo.json

```diff
@@ -0,0 +1,21 @@
+{
+  "providerId": "bing.com",
+  "providerName": "Microsoft Bing",
+  "serviceId": "bingwebmasterdpoo",
+  "serviceName": "Domain proof of ownership with Bing Webmaster services",
```

Comment:
Wonder if you want something shorter here.  Note: Right now this string is displayed to the user, and there is not a mechanism in the specification for localization.  So you probably want something simpler....

Reply:
👍 for this one. A shorter name and something that would fit better even in localized texts would be a way better.

Reply:
Agree, changed.